source/agora/cli/multi/main.d

```diff
@@ -104,9 +104,9 @@ private int main (string[] args)
         }
     }
 
-    Node[] nodes;
+    Validator[] nodes;
     foreach (const ref config; configs)
-        nodes ~= runNode(config);
+        nodes ~= runNode!Validator(config);
```

Comment:
I think this is wrong. The configs that are passed can be both Validators and FullNodes. So the code will have to check each config and spawn each type. accordingly.

Reply:
The following files are also required when testing modifications.
ci/single_binary_test.d.
I think it needs to be corrected.

Reply:
applied the code checks each config and spawn each type.